src/platform/apollolake/include/platform/platform.h

```diff
@@ -186,6 +186,10 @@ static inline void platform_panic(uint32_t p)
 #define platform_trace_point(__x) \
 	mailbox_sw_reg_write(SRAM_REG_FW_TRACEP, (__x))
 
+#define PLATFORM_XTAL_OSCILLATOR_CLK 19200000
```

Comment:
Those clock defines better to be placed in platform/clk.h.

Reply:
+1 - clk.h seems to be more appropriate location. The values could be obtained from the clk API by the driver, no need to include the header and make any max- calculations inside the driver. Or should the SSP clock be managed by the SSP driver rather than the central clk unit to make the code easier configurable by the CONFIG_ settings?

Reply:
I think this clk is already defined in 
https://github.com/thesofproject/sof/blob/052942c8636e0b7611e8d13d8cc97ba2966a3b4e/src/platform/apollolake/include/platform/clk-map.h#L40-L42
Maybe we need some unification of this clk defines with the timers.

Reply:
I've added ssp clk definitions to platform/clk-map.h